src/wattsi.pas

```diff
@@ -518,6 +523,16 @@ procedure AddMDNBox(const MDNBox: TElement;
       for BrowserID in MDNBrowsersForMobileDevices do
          ProcessBrowserData(BrowserID, MDNSupport[BrowserID], SupportTable,
                             Document);
+      if (Assigned(MDNData['caniuse'])) then
+      begin
+         SupportTable.AppendChild(E(eHR));
+         CanIUseRow := E(eSpan, ['data-x', '', 'class', 'caniuse'], Document);
+         CanIUseRow.AppendChild(E(eSpan, ['data-x', ''],
+            [E(eA, ['href',
+               kCanIUseURLBase + UTF8String(MDNData['caniuse']['feature'])],
+               Document, [T(MDNData['caniuse']['title'], Document)])]));
```

Comment:
Hmm. I wonder if the title should just be "caniuse.com" or "See on caniuse.com" or something? Now we have two "competing" titles in the same popup, e.g. 
![image](https://user-images.githubusercontent.com/617481/84537344-8b0e7b80-acbd-11ea-8246-81acf5562b8f.png)

(ignore the second box on top, I know that will not be there)

Reply:
> Now we have two "competing" titles in the same popup

Yeah, I thought the same. It’s suboptimal.

> I wonder if the title should just be "caniuse.com" or "See on caniuse.com" or something? 

I thought about that too, but to me it seems suboptimal to repeat the "caniuse.com" or "See on caniuse.com" text.

So maybe it’d be better to reduce it even further. Lemme know what do you think of this idea: Rather than having an entire row in the MDN box for the Caniuse link at the end of the box, we instead just put a Caniuse hyperlinked icon after the MDN article slug, on the same line.

We could them move the Caniuse page title into tooltip text that’d be shown when users hover over the icon.

Reply:
I slightly prefer the repetitive text, but I don't feel strongly. I think it'd be a bit more mobile-friendly than an icon with hovertext (both in touch-target size and readability), and I worry a bit about smooshing the icon next to the title text making it look weird. But I'm OK to trust your judgment.

Reply:
> I slightly prefer the repetitive text, but I don't feel strongly. I think it'd be a bit more mobile-friendly than an icon with hovertext (both in touch-target size and readability), and I worry a bit about smooshing the icon next to the title text making it look weird.

Yeah, agreed — all good points

> I wonder if the title should just be "caniuse.com" or "See on caniuse.com" or something? 

I went with “caniuse.com table” — since “table” is what caniuse.com seems to use itself (rather than “page” or whatever)